### PR TITLE
Fix strpos check

### PR DESCRIPTION
**Pull request by gallart**

### CODE REVIEW

File: scan.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ function detect_shell($filecontent)
     global $shells;
 
     foreach ($shells as $shell) {
-        if (strpos(implode($filecontent), trim($shell))) {
+        if (strpos(implode($filecontent), trim($shell)) !== FALSE) {
```
Comment (soullivaneuh) [inline scan.php]: `false !== strpos(implode($filecontent), trim($shell))`

* Yoda condition
* lowercase constant

File: scan.php [hunk 2]
```diff
@@ -75,7 +75,7 @@ function in_whitelist($filename)
     global $whitelist;
 
     foreach ($whitelist as $wl) {
-        if (strpos($filename, trim($wl))) {
+        if (strpos($filename, trim($wl)) !== FALSE) {
```
Comment (soullivaneuh) [inline scan.php]: Same as the other.

**Review by soullivaneuh — CHANGES_REQUESTED** [review]

File: scan.php (hunk 1 above)
Comment (gallart) [reply scan.php]: fixed

File: scan.php (hunk 2 above)
Comment (gallart) [reply scan.php]: fixed

**Review by soullivaneuh — APPROVED** [review]

**Pull request merged**